spec/features/admin_user_create_events_spec.rb

```diff
@@ -24,6 +25,27 @@
         expect(page).to have_content 'New Events'
       end
     end
+
+    context 'when event exists' do
+      before :each do
+        visit '/admin'
+        click_link 'New event'
+        fill_event_form(event_params)
```

Comment:
Let's create an event with the Event factory here instead of filling the form on each step. It would be faster.
e.g. `let!(:event) { create(:event) } `